swift/ObjectHash/ObjectHash.swift

```diff
@@ -1,21 +1,19 @@
-import CryptoSwift
-
 public typealias Byte = UInt8
 public typealias BytesArray = [Byte]
 public typealias Object = [String: Any]
 
 public class ObjectHash {
     public internal(set) var hash: BytesArray = []
-    private var digester = SHA2(variant: .sha256)
-    
+    private var digester = Digester(algorithm: .sha256)
```

Comment:
oh, i see now

Reply:
I would suggest abstract ObjectHash from digester, and pass Digester on ObjectaHash construction or via settings.